syntax = "proto3";
package LPMsg;

message ReqLogin
{
	string account = 1;
	string password = 2;
}

message RoomInfo
{
	string		ip = 1;
	int32		port = 2;
	int32		uid = 3; 
	int32		key = 4;
}

message CmLogin
{
	int32	uid = 1;
	int32	key = 2;
}
